packages/design-system/src/components/text-field.tsx

```diff
@@ -55,6 +55,14 @@ export const TextFieldIconButton = styled(
             color: "white",
           },
         },
+        highlight: {
```

Comment:
sure ,but then later you also need the orange one, for inherited values

Reply:
We should rather use that icon button and have its own states in CSS input

styled(TextFieldIconButton, {...specific stuff for style panel})